content/blog/2020/11/2020-11-11-jira-upgrade-for-the-jenkins-project.adoc

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Jira upgrade for the Jenkins project"
+tags:
+- jenkins
+- infrastructure
+- jira
+author: markewaite
+description: "The Linux Foundation is now hosting Jira 8.13 for the Jenkins project"
+opengraph:
+  image: /images/post-images/2020-11-11-jira-upgrade-for-the-jenkins-project/jira-upgrade-for-the-jenkins-project.png
+---
+
+image::/images/post-images/2020-11-11-jira-upgrade-for-the-jenkins-project/jira-logo.png[float="right",align="center"]
+
+The Jenkins project has used Jira to track issues for many years.
+Jenkins core, Jenkins modules, Jenkins infrastructure, and many Jenkins plugins manage their issue reports with our link:https://issues.jenkins.io/[Jira server].
+
+Jira helps the Jenkins project manage issues and tasks related to over 250{nbsp}000 Jenkins installations.
+It tracks bugs, enhancement requests, tasks, and security issues.
+It is used regularly by users around the world.
+
+We're grateful for the long-standing contribution that Atlassian provides by donating the Jira license to the Jenkins project.
+We're grateful to the link:https://osuosl.org/[Oregon State University Open Source Lab] for their donation of equipment and bandwidth to host the server.
+
+== Upgrade Timeline
+
+We were running Jira 7.13 and had been managing that installation for a few years.
+Atlassian announced that Jira 7.13 would end its support life November 28, 2020.
+We needed to upgrade from Jira 7.13 to a more recent version of Jira.
+
+During the summer of 2020, Tracy Miranda discovered that the Linux Foundation offers Jira hosting for open source projects.
+As part of our membership in the Continuous Delivery Foundation, a Linux Foundation initiative, we could use their project services team to manage our Jira server.
+We decided to move from hosting our own Jira server to having the Jira experts at the Linux Foundation host it.
+
+The upgrade timeline looked like this:
+
+* November 2019 - Infrastructure team begins discussions about the November 2020 end of support for Jira 7.13
```

Comment:
This date seems weird to me

Reply:
November 2019 was the first reference I found in infra team meeting notes to the end of support for Jira 7.13.  Happy to change it to something else or remove it if the timeline is not helpful.